src/garage/torch/algos/vpg.py

```diff
@@ -180,10 +180,11 @@ def _add_padding(self, tensor, length):
         return F.pad(tensor, (0, padding_length))
 
     def _get_policy_entropy(self, obs):
-        policy_entropy = self.policy.get_entropy(obs).sum()
-
         if self.stop_entropy_gradient:
-            policy_entropy.requires_grad = False
+            with torch.no_grad():
+                policy_entropy = self.policy.get_entropy(obs)
```

Comment:
please ensure that all class members are private (i.e. `_policy`) unless they are part of the public interface (in which case they should usually be a property, and not an attribute).

please apply this to all of the class members, not just policy.

Reply:
policy, baseline, discount, max_path_length, and n_samples are in the parent class (BatchPolopt). and BatchPolopt class is used in other tensorflow algorithms (ex npo)

But I can change every other class members in the VPG into private.
Is it okay to refactor BatchPolopt and every other related classes?

Reply:
ah -- no don't touch that one if it's going to lead you outside the torch tree.

Reply:
okay, just fixed variables inside the VPG class